shell/control-center.c

```diff
@@ -70,6 +70,8 @@ static GSList* get_actions_list(void)
 
 		if ((action->item = load_desktop_item_from_unknown(temp[1])) == NULL)
 		{
+			g_free ((char*)action->name);
```

Comment:
Perhaps `name` should not be const in first place?

Reply:
thanks for the tip